packages/libsolv/libsolv.spec

```diff
@@ -232,10 +235,16 @@ set -ex
   -DPYTHON_EXECUTABLE=%{__python2}               \
 %if %{with python3_bindings}
   -DENABLE_PYTHON3=ON                            \
-  -DPYTHON3_EXECUTABLE=%{__python3}              \
+  -DPYTHON_EXECUTABLE=/usr/bin/python3.11        \
+  -DPYTHON_LIBRARY=/usr/lib64/libpython3.11.so.1.0 \
+  -DPYTHON_INCLUDE_DIR=/usr/include/python3.11   \
+  -DPython_ADDITIONAL_VERSIONS=3.11 \               
 %endif
 %else
   -DPYTHON_EXECUTABLE=%{__python3}               \
+  -DPYTHON_LIBRARY=/usr/lib64/libpython3.11.so.1.0 \
+  -DPYTHON_INCLUDE_DIR=/usr/include/python3.11   \
+  -DPython_ADDITIONAL_VERSIONS=3.11 \
```

Comment:
This one jumps out, are you sure this is supposed to be lowercase? And I don't see `ADDITIONAL` in the libsolv source code at all.

Reply:
Maybe this is a flag that is not used anymore, let me point fingers at @evgeni he probably knows